### ISSUE TITLE
about accuracy on cfp-fp and agedb-30 in the arxiv paper

### ISSUE THREAD

**Issue by rocketbear** [deepinsight/insightface#147]
According to the numbers in the paper, it seems that models trained on VGG2 have higher accuracy on CFP-FP than models trained on MS1M, while the relation is reversed for AgeDB-30. For example:
Table 5, SE-LResNet50E-IR, softmax@VGG2, using WD=5e-4, 
Accuracy on CFP-FP=96.82%, accuracy on AgeDB-30=93.83%

Table 7, LResNet100E-IR@MS1M, ArcFace m=0.5:
Accuracy on CFP-FP=94.04, accuracy on AgeDB-30=98.08%.

Does this mean that there is something alike between VGG2 and CFP-FP, such that a model trained on VGG2 already knows something about CFP-FP and performs better than that trained on MS1M? What characteristics of VGG2 may cause this phenomenon? Identity overlap or that VGG2 contains much more varied pose than MS1M?

**Comment by nttstar** [comment]
`Identity overlap or that VGG2 contains much more varied pose than MS1M?`
Maybe both.
